website/docs/reference/warehouse-profiles/layer-profile.md

```diff
@@ -2,100 +2,160 @@
 title: "Layer Profile"
 ---
 
-
 ## Overview of dbt-layer
-**Maintained by:** Layer      
-**Authors:** Mehmet Ecevit  
-**Source:** [Github](https://github.com/layerai/dbt-layer)  
-**Core version:** `v1.0` and newer      
-**dbt Cloud:** Not Supported      
-**dbt Slack channel** [#db-layer](https://slack.com)
 
-[![dbt-synapse stars](https://img.shields.io/github/stars/layerai/dbt-adapters?style=for-the-badge)](https://github.com/layerai/dbt-adapters)
-[![latest version on PyPI](https://img.shields.io/pypi/v/dbt-layer-bigquery?style=for-the-badge)](https://pypi.org/project/dbt-layer-bigquery)
+**Maintained by:** [Layer](https://layer.ai)
+**Authors:** [Layer](https://layer.ai)
+**Source:** [Github](https://github.com/layerai/dbt-layer)
+**Core version:** `v1.2` and newer
+**dbt Cloud:** Not Supported
+**dbt Slack channel** [#tools-layer](https://slack.com)
+
+[![dbt-layer stars](https://img.shields.io/github/stars/layerai/dbt-adapters?style=for-the-badge)](https://github.com/layerai/dbt-layer)
+[![latest version on PyPI](https://img.shields.io/pypi/v/dbt-layer?style=for-the-badge)](https://pypi.org/project/dbt-layer)
 
-To immediately start using Layer inside your dbt DAG, install the Layer dbt Adapter for BigQuery.
+To start using Layer inside your dbt DAG, get a Layer API key and install the Layer dbt Adapter for your warehouse:
 
 ```
-pip install dbt-layer-bigquery
+pip install dbt-layer[<warehouse>]
 ```
 
-You don't need to install dbt separately. Installing `dbt-layer` will also install `dbt-core` and `dbt-bigquery`.
+## Setup
 
+### Layer
 
-### Profile Configuration
+To access Layer, you will need a Layer API key. To get this:
 
-Layer Bigquery targets should be set up using the following sections in your `profiles.yml` file.
-#### Layer Authentication
-Add your `layer_api_key` to your `profiles.yaml` to authenticate with Layer. To get your Layer API Key:
 - First, [create your free Layer account](https://app.layer.ai/login?returnTo=%2Fgetting-started).
-- Go to [app.layer.ai](https://app.layer.ai) > **Settings** (Cog Icon by your profile photo) > **Developer** > **Create API key** to get your Layer API Key.
+- Go to [app.layer.ai](https://app.layer.ai) > **Settings** (Cog Icon by your profile photo) > **Developer** > **Create API key**
+
+Add this as the `layer_api_key` in your `profiles.yaml`.
+
+## Setup Your Warehouse
+
+Once you have Layer set up, choose one of the following supported warehouses:
+
+### BigQuery
```

Comment:
love to see support for another adapter here. I'd be curious if this can be generalized to pertain to both adapters rather than written out as two separate sections...

alternatively, there is a way to combine into one section and have tabs for each of Snowflake and BigQuery. here's an example snippet of how to pull off tabs to get the below result
https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/docs/reference/warehouse-profiles/mssql-profile.md?plain=1#L46-L97
![image](https://user-images.githubusercontent.com/8158673/184410560-c59d499d-2eaf-4456-b5b9-289509f9c428.png)